.github/actions/release/action.yaml

```diff
@@ -38,10 +38,5 @@ runs:
       env:
         GH_TOKEN: ${{ inputs.gh_token }}
         GITHUB_TOKEN: ${{ inputs.gh_token }}
-      run: npx nx affected --base=last-release --parallel=1 --target=version --postTargets=npm,github --trackDeps --exclude=@redhat-cloud-services/CLIENTNAME-client
-    - name: Tag last-release
-      shell: bash
-      run: |
-        git tag -f last-release
-        git push origin last-release --force
+      run: npx nx affected --base=HEAD~1 --parallel=1 --target=version --postTargets=npm,github --trackDeps
```

Comment:
This fixes the issue where the affected packages were being compared to an outdated base